focus-areas/what/technical-fork.md

```diff
@@ -0,0 +1,49 @@
+###This metric is a release candidate To comment on this metric please see Issue #xx Following a comment period, this metric will be included in the next regular release.
+
```

Comment:
Can you create the issue and update this with the issue number before we merge it?

I also think this is missing some punctuation. It looks like there should be a period after candidate and another one after the issue number?

Reply:
Updated.

Reply:
Much better, thank you!